Fix: linter should force to put boolean value instead of omitting tru…

VladimirKopychev-udemy opened the pull request:
##### *To:*
@udemy/team-f 
@kevinzang 
@jjinux 

##### *What:*
It's recommended by official ReactJS doc to always put boolean values for properties
https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true
So we need to change linter rule to check it

VladimirKopychev-udemy committed:
fix: react/jsx-boolean-value - put "always" option instead of "never".

BREAKING CHANGE: explicit true is now required for boolean prop values.

kevinzang commented:
@VladimirKopychev-udemy code change LGTM! I have a suggestion for your commit message though: can you add something like "BREAKING CHANGE: explicit true is now required for boolean prop values" to the commit *description*? Your commit title is fine.

I mean something like https://github.com/udemy/js-tooling/pull/51/commits/1655e2f90fc3fa4943bd7ed85a7fd61528791fcb

Notice that the description has "BREAKING CHANGE: Upgrade ESLint to ^5.9.0 and require node >=8.12.0 as an engine.". When lerna sees "BREAKING CHANGE" at the beginning of a commit description, it adds that to the CHANGELOG when you run `lerna publish`. This helps with documentation.

VladimirKopychev-udemy commented:
Commit description is done @kevinzang 
Thanks for the guidelines
👍

kevinzang reviewed (approved):
@VladimirKopychev-udemy LGTM!

VladimirKopychev-udemy committed:
Publish

 - babel-polyfill-udemy-website@9.0.3
 - babel-preset-udemy-website@10.0.3
 - eslint-config-udemy-react-addons@10.0.0
 - eslint-config-udemy-website@12.0.2
